packages/waku/src/lib/config.ts

```diff
@@ -2,6 +2,7 @@ import fs from 'node:fs';
 import path from 'node:path';
 
 import type { Config } from '../config.js';
+import { normalizePath } from 'vite';
```

Comment:
This file is used in production. So, we can't depend on `vite`.

Maybe, we need to take `command` argument in `resolveConfig` function below, and only if it's `'dev'` , do `import('vite')`. We do this in various places.